<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VirtualNumpad</class>
 <widget class="QDialog" name="VirtualNumpad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>VirtualNumpad</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
	background-color: black;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLineEdit" name="weightEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>800</width>
       <height>150</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>800</width>
       <height>240</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit#weightEdit{
	background-color: white;
	color: black;
	font-size: 24px;
}</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>Weight</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="clearButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>55</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>55</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	font-weight: bold;
	color: white;
	background-color: rgb(36, 36, 36);
	border-radius: 5px;
	font-size: 11px;

}

QPushButton:hover{
	background-color: rgb(207, 207, 207)
}</string>
         </property>
         <property name="text">
          <string>CLR</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="leftArrowButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>55</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>55</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	font-weight: bold;
	color: white;
	background-color: rgb(36, 36, 36);
	border-radius: 5px;
	font-size: 11px;
}

QPushButton:hover{
	background-color: rgb(207, 207, 207)
}</string>
         </property>
         <property name="text">
          <string>LEFT</string>
         </property>
         <property name="iconSize">
          <size>
           <width>70</width>
           <height>51</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="rightArrowButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>55</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>55</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	font-weight: bold;
	color: white;
	background-color: rgb(36, 36, 36);
	border-radius: 5px;
	font-size: 11px;
}

QPushButton:hover{
	background-color: rgb(207, 207, 207)
}</string>
         </property>
         <property name="text">
          <string>RIGHT</string>
         </property>
         <property name="iconSize">
          <size>
           <width>70</width>
           <height>51</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="goBackButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>55</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>55</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	font-weight: bold;
	color: white;
	background-color: rgb(36, 36, 36);
	border-radius: 5px;
	font-size: 11px;
}

QPushButton:hover{
	background-color: rgb(207, 207, 207)
}</string>
         </property>
         <property name="text">
          <string>BACK</string>
         </property>
         <property name="iconSize">
          <size>
           <width>70</width>
           <height>51</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="sevenButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>55</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>55</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	font-size: 15px;
	font-weight: bold;
	color: white;
	background-color: rgb(79, 79, 79);
	border-radius: 5px;
}

QPushButton:hover{
	background-color: rgb(207, 207, 207)
}</string>
         </property>
         <property name="text">
          <string>7</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="eightButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>55</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>55</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	font-size: 15px;
	font-weight: bold;
	color: white;
	background-color: rgb(79, 79, 79);
	border-radius: 5px;
}

QPushButton:hover{
	background-color: rgb(207, 207, 207)
}</string>
         </property>
         <property name="text">
          <string>8</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="nineButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>55</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>55</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	font-size: 15px;
	font-weight: bold;
	color: white;
	background-color: rgb(79, 79, 79);
	border-radius: 5px;
}

QPushButton:hover{
	background-color: rgb(207, 207, 207)
}</string>
         </property>
         <property name="text">
          <string>9</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3" rowspan="2">
        <widget class="QPushButton" name="deleteButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>120</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>120</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	font-weight: bold;
	color: white;
	background-color: rgb(36, 36, 36);
	border-radius: 5px;
	font-size: 11px;
}

QPushButton:hover{
	background-color: rgb(207, 207, 207)
}</string>
         </property>
         <property name="text">
          <string>DEL</string>
         </property>
         <property name="iconSize">
          <size>
           <width>60</width>
           <height>52</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="fourButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>55</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>55</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	font-size: 15px;
	font-weight: bold;
	color: white;
	background-color: rgb(79, 79, 79);
	border-radius: 5px;
}

QPushButton:hover{
	background-color: rgb(207, 207, 207)
}</string>
         </property>
         <property name="text">
          <string>4</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="fiveButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>55</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>55</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	font-size: 15px;
	font-weight: bold;
	color: white;
	background-color: rgb(79, 79, 79);
	border-radius: 5px;
}

QPushButton:hover{
	background-color: rgb(207, 207, 207)
}</string>
         </property>
         <property name="text">
          <string>5</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="sixButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>55</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>55</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	font-size: 15px;
	font-weight: bold;
	color: white;
	background-color: rgb(79, 79, 79);
	border-radius: 5px;
}

QPushButton:hover{
	background-color: rgb(207, 207, 207)
}</string>
         </property>
         <property name="text">
          <string>6</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="oneButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>55</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>55</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	font-size: 15px;
	font-weight: bold;
	color: white;
	background-color: rgb(79, 79, 79);
	border-radius: 5px;
}

QPushButton:hover{
	background-color: rgb(207, 207, 207)
}</string>
         </property>
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="twoButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>55</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>55</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	font-size: 15px;
	font-weight: bold;
	color: white;
	background-color: rgb(79, 79, 79);
	border-radius: 5px;
}

QPushButton:hover{
	background-color: rgb(207, 207, 207)
}</string>
         </property>
         <property name="text">
          <string>2</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="threeButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>55</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>55</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	font-size: 15px;
	font-weight: bold;
	color: white;
	background-color: rgb(79, 79, 79);
	border-radius: 5px;
}

QPushButton:hover{
	background-color: rgb(207, 207, 207)
}</string>
         </property>
         <property name="text">
          <string>3</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3" rowspan="2">
        <widget class="QPushButton" name="enterButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>120</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>120</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	font-weight: bold;
	color: white;
	background-color: rgb(36, 36, 36);
	border-radius: 5px;
	font-size: 11px;
}

QPushButton:hover{
	background-color: rgb(207, 207, 207)
}</string>
         </property>
         <property name="text">
          <string>ENTER</string>
         </property>
         <property name="iconSize">
          <size>
           <width>100</width>
           <height>51</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QPushButton" name="zeroButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>114</width>
           <height>55</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>106</width>
           <height>55</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	font-size: 15px;
	font-weight: bold;
	color: white;
	background-color: rgb(79, 79, 79);
	border-radius: 5px;
}

QPushButton:hover{
	background-color: rgb(207, 207, 207)
}</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QPushButton" name="periodButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>55</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>55</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	font-size: 15px;
	font-weight: bold;
	color: white;
	background-color: rgb(79, 79, 79);
	border-radius: 5px;
}

QPushButton:hover{
	background-color: rgb(207, 207, 207)
}</string>
         </property>
         <property name="text">
          <string>.</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
